addons/event_sale/models/event_registration.py

```diff
@@ -131,6 +131,8 @@ def _sale_order_ticket_type_change_notify(self, new_event_ticket):
 
     def _get_registration_summary(self):
         res = super(EventRegistration, self)._get_registration_summary()
+        if not self.sale_status:
+            self._compute_registration_status()
```

Comment:
Anyway you shouldn't call compute methods manually for issues like this. It only makes sense to call them manually if a correct dependency isn't possible.